[2024-06-03T22:32:07.628+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T22:32:07.643+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-02-01T00:00:00+00:00 [queued]>
[2024-06-03T22:32:07.649+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-02-01T00:00:00+00:00 [queued]>
[2024-06-03T22:32:07.650+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T22:32:07.660+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2022-02-01 00:00:00+00:00
[2024-06-03T22:32:07.668+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=31157) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T22:32:07.670+0000] {standard_task_runner.py:63} INFO - Started process 31159 to run task
[2024-06-03T22:32:07.670+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2022-02-01T00:00:00+00:00', '--job-id', '139', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpquoqnhsy']
[2024-06-03T22:32:07.671+0000] {standard_task_runner.py:91} INFO - Job 139: Subtask blunders_etl
[2024-06-03T22:32:07.705+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2022-02-01T00:00:00+00:00 [running]> on host f55fff24b127
[2024-06-03T22:32:07.771+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2022-02-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-01T00:00:00+00:00'
[2024-06-03T22:32:07.772+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T22:32:07.774+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T22:32:07.780+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-03T22:47:25.862+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T22:47:25.863+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T22:47:25.871+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2022-02-01T00:00:00+00:00, execution_date=20220201T000000, start_date=20240603T223207, end_date=20240603T224725
[2024-06-03T22:47:25.897+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T22:47:25.909+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T22:47:25.911+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T19:55:51.940+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T19:55:51.955+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-02-01T00:00:00+00:00 [queued]>
[2024-06-08T19:55:51.962+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-02-01T00:00:00+00:00 [queued]>
[2024-06-08T19:55:51.962+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T19:55:51.972+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2022-02-01 00:00:00+00:00
[2024-06-08T19:55:51.979+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=31422) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T19:55:51.980+0000] {standard_task_runner.py:63} INFO - Started process 31424 to run task
[2024-06-08T19:55:51.981+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2022-02-01T00:00:00+00:00', '--job-id', '315', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpc10o4yig']
[2024-06-08T19:55:51.981+0000] {standard_task_runner.py:91} INFO - Job 315: Subtask blunders_etl
[2024-06-08T19:55:52.016+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2022-02-01T00:00:00+00:00 [running]> on host bf0ade909435
[2024-06-08T19:55:52.074+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2022-02-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-01T00:00:00+00:00'
[2024-06-08T19:55:52.075+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T19:55:52.076+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T19:55:52.082+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T20:12:49.627+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T20:12:49.628+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T20:12:49.638+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2022-02-01T00:00:00+00:00, execution_date=20220201T000000, start_date=20240608T195551, end_date=20240608T201249
[2024-06-08T20:12:49.682+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T20:12:49.697+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-08T20:12:49.699+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-08T01:55:17.995+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-08T01:55:18.009+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-02-01T00:00:00+00:00 [queued]>
[2024-07-08T01:55:18.016+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2022-02-01T00:00:00+00:00 [queued]>
[2024-07-08T01:55:18.016+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-08T01:55:18.025+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2022-02-01 00:00:00+00:00
[2024-07-08T01:55:18.032+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=32431) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-08T01:55:18.034+0000] {standard_task_runner.py:63} INFO - Started process 32433 to run task
[2024-07-08T01:55:18.034+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2022-02-01T00:00:00+00:00', '--job-id', '705', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpso58r5vc']
[2024-07-08T01:55:18.035+0000] {standard_task_runner.py:91} INFO - Job 705: Subtask blunders_etl
[2024-07-08T01:55:18.073+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2022-02-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-08T01:55:18.136+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2022-02-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-02-01T00:00:00+00:00'
[2024-07-08T01:55:18.137+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-08T01:55:18.138+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-08T01:55:18.144+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-08T02:10:08.838+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-08T02:10:08.839+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-08T02:10:08.846+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2022-02-01T00:00:00+00:00, execution_date=20220201T000000, start_date=20240708T015518, end_date=20240708T021008
[2024-07-08T02:10:08.896+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-08T02:10:08.912+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-08T02:10:08.913+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
